Src/FluentAssertions/Execution/DefaultAssertionStrategy.cs

```diff
@@ -1,8 +1,10 @@
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using FluentAssertions.Common;
 
 namespace FluentAssertions.Execution;
 
+[ExcludeFromCodeCoverage]
 internal class DefaultAssertionStrategy : IAssertionStrategy
 {
```

Comment:
Maybe there are a few other places with default implementations, which could also be excluded?